build-gluster-org/scripts/flake8.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+RESULT="$WORKSPACE/flake8"
+mkdir $RESULT
+
+./autogen.sh
+./configure --enable-debug --enable-gnfs --silent
+
+# create and activate virtual env
+python3 -m venv env
+. env/bin/activate
+
+#install flake8
+pip install -I flake8
+
+# run flake8
+find . -path './env' -prune -o -name '*.py' -print | xargs flake8 > "$RESULT/flake8-check.txt"
+FLAKE_COUNT="$(wc -l < $RESULT/flake8-check.txt)"
+
+#fail build if there's any flake8 related issues
+if [ $FLAKE_COUNT -gt 0 ]; then
+  echo ""
+  echo "========================================================="
+  echo "              Result of python linter"
+  echo "         Number of flake8 issues: ${FLAKE_COUNT}"
+  echo "========================================================="
```

Comment:
There should be return code `exit 1`. Else it will go unnoticed.

Reply:
Yes! but again won't it block PRs?

Reply:
yeah, I think that's why we were looking at using warnings-ng later, but in the meantime, since the codebase is not clean, we didn't want to block.